VOCABULARY-CREATION-GUIDE.md

```diff
@@ -2,4 +2,33 @@
 
 Thing Description Forms can be extended with annotations on how to activate affordances.
 These annotations are typically from protocol vocabularies that need to exist separately from the binding template document.
-This document gives guidance how such vocabularies can be created and how they should be used together with the templates and tools provided.
\ No newline at end of file
+This document gives guidance how such vocabularies can be created and how they should be used together with the templates and tools provided.
+
+## Components of a Binding Template
+
+Besides the actual binding template document that is to be used by implementors
+(the `index.html` file), there are several additional companion documents which
+enable the binding to be used in practice.
+To enable TD consumers to work with the added vocabulary, the definition of a
```

Comment:
To enable TD consumers understand the reused vocabularies,...

Reply:
Thank you for your feedback, @mahdanoura! Where possible, I will turn your comments into GitHub suggestions like the one below so that they can be directly applied from the PR user interface.

```suggestion 
To enable TD consumers to understand the reused vocabularies, the definition of a
```